samples/plugins/postgres_jdbc/manifest.xml

```diff
@@ -30,7 +30,8 @@
       <customization name="CAP_QUERY_TIME_REQUIRES_CAST" value="yes"/>
     </customizations>
   </connection-customization>
-  <connection-dialog file='connection-dialog.tcd'/>
-  <connection-resolver file="connectionResolver.tdr"/>
+  <connection-fields file='connectionFields.xml'/>
+  <connection-metadata file='connectionMetadata.xml'/>
+  <connection-resolver file="connectionResolver.xml"/>
```

Comment:
I want us to do this tdr to xml change, but I think we'll need to update the docs more to do it. Tracked in 1072286, but not scheduled currently (we could).